submissions/Barasii/a-tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,86 @@
+class Inhabitant {
+   constructor(species, name, gender, legs, hands, saying, friends) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.hands = hands;
+      this.saying = saying;
+      this.friends = [];
+   }
+
+   toString() {
```

Comment:
Nice. It's an extra point to your score.

Reply:
@OleksiyRudenko,
thank you for your helping